docs/learn/platformio.md

```diff
@@ -0,0 +1,79 @@
+
+![!VS Code Extension](https://www.v1engineering.com/wp-content/uploads/2019/12/Extension.jpg){: width="400"}
+
+This will pop up a search window. Type in "PlatformIO". It should be the first result, now just click the green install button.
+
+Now the difficult part....you have to wait for it to finish. It can take up to 15 minutes on an older machine.
+
+![!VS Code Waiting](https://www.v1engineering.com/wp-content/uploads/2019/12/Still_installing.jpg){: width="400"}
+
+At the very bottom of the VS Code window you will see a purple bar and the installing icon and message, touch nothing while it works. When it is done it will pop up a restart button.
+
+![!VS Code restart](https://www.v1engineering.com/wp-content/uploads/2019/12/restart.jpg){: width="400"}
+
+Congratulations you are done with the "hard part"!
+
+---
+
+## Using PlatformIO
+
+Once you have PlatformIO installed there is really nothing else that needs to be added. Now you can load your firmware of choice and flash or compile.
+
+### Load Your Firmware
+
+You should already have downloaded and **Unzipped** the firmware for your board. If you are unsure of that step have a look [here](../electronics/marlin-firmware.md).
+
+In the top left of the VS Code window select "File", then "Open Folder".
+
+![!VS Code Open Folder](https://www.v1engineering.com/wp-content/uploads/2019/12/open-folder.png){: width="400"}
+
+You can select the entire folder you have unzipped. If that is unclear - You need the entire folder that contains "platformio.ini" and the "marlin" folder. Here is a picture of what that looks like. The name will vary as the versions progress, bugfix 2.0.x is another common one.
+
+![!VS Code Folder](https://www.v1engineering.com/wp-content/uploads/2019/12/folder-selected.jpg){: width="400"}
+
+One you open that folder it will show in the window and look very similar to the picture below. **Notice** at the very bottom, the blue banner shows a "Index Rebuild" icon let that finish before moving to the next step.
+
+![!VS Code Folder Open](https://www.v1engineering.com/wp-content/uploads/2019/12/folder-open.jpg){: width="400"}
+
+### Build
+
+At this point it is ready to go. Most boards can be plugged in a "flashed"  or some boards just want you to compile the firmware and load it onto an SD card. Only the last step differs.
+
+Hit the check mark to build (or compile) the firmware. Most items will say "ignored" as it only builds for the one specified board.
+
+![!VS Code Compile](https://www.v1engineering.com/wp-content/uploads/2019/12/build.png){: width="400"}
+
+At this point you should get a message that says "1 succeeded". If you do not the firmwre has build issues and they need to be resolved. 
+
+The next step is to get it onto you board, or if you are sharing your firmware you can skip to the Compile step.
+
+**Flashing** (most common)
```

Comment:
Do we need to mention anything about which board supports flashing and which ones support using the .bin file? 

The SKR board was much easier to flash by copying the .bin file over, and obviously the atmega boards need to be flashed. If you enabled the SD card support for the SKR, then it wouldn't actually be able to flash it from platformio.